rr_iarrc/src/laplacian_line_detection/laplacian_line_detector.cpp

```diff
@@ -77,7 +77,7 @@ int main(int argc, char** argv) {
 
     nhp.param("subscription_node", subscription_node, std::string("/camera/image_color_rect"));
 
-    pub = nh.advertise<sensor_msgs::Image>("/Combined_lines", 1); //test publish of image
+    pub = nh.advertise<sensor_msgs::Image>("/lines_detection_img", 1); //test publish of image
```

Comment:
We may want to come up with a better naming patter for our line detection topics. Maybe `/lines/<thing>` since we have a couple secondary debug ones to go along with the final lines output.

Reply:
Also remove the comment since that is the final image. Pretty sure that is from people copying and pasting from my example/test image node haha.